.github/actions/paths-filter/action.yml

```diff
@@ -17,6 +17,13 @@ outputs:
         github.event_name == 'schedule' ||
         steps.filter-common.outputs.github-actions-change == 'true'
       }}
+  commitlint:
+    description: Output whether commit messages should be linted based on GitHub event and files changed
+    value: >-
+      ${{
+        github.event_name == 'pull_request' &&
+        !contains(github.event.pull_request.labels.*.name, 'ci:ignore-commitlint')
+      }}
```

Comment:
👍 Great idea with the label

🔧 Let's document this in:
- Contributing guide
- If possible in the commitlint error message
  - or add a workflow run summary item explaining this

Reply:
> 🔧 Let's document this in:

For sure, I'll look into that next so _if_ people need it they can find it.

Also the label itself needs a description still.